comps/dataprep/redis/langchain/docker/docker-compose-dataprep-redis.yaml

```diff
@@ -21,6 +34,8 @@ services:
       no_proxy: ${no_proxy}
       http_proxy: ${http_proxy}
       https_proxy: ${https_proxy}
+      REDIS_HOST: ${REDIS_HOST}
```

Comment:
the `TEI_ENDPOINT` is not set here？we need this env in the code https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/redis/langchain/prepare_doc_redis.py#L36